vars/asfMavenTlpPlgnBuild.groovy

```diff
@@ -40,12 +40,12 @@ def call(Map params = [:]) {
     // minimum, LTS, current and next ea
     def jdks = params.containsKey('jdks') ? params.jdks : params.containsKey('jdk') ? params.jdk : ['8','11','17']
     def jdkMin = jdks[0];
-    def mavens = params.containsKey('maven') ? params.maven : ['3.6.x', '3.8.x']
+    def mavens = params.containsKey('maven') ? params.maven : ['3.6.x', '3.x.x']
```

Comment:
the naming is really weird.. shouldn’t we have something such `3.9.x`?

Reply:
can be 😄

Reply:
done